### ISSUE TITLE
Language / Location setting

### ISSUE THREAD

**Issue by o86** [egirault/googleplay-api#7]
Since the results vary from location to location, it would be an advantage to have the option to over-steer your configuration of France/French.

If not with an option, at leas with a global variable in the config.py. 

What's your idea about that?

**Comment by egirault** [comment]
Yes indeed. It would be a great thing to be able to request the API from
different countries / mobile providers. I just did not have the time to
handle this yet, but hopefully I will soon.
Anyway, feel free to initate this feature if you are motivated ;)

2012/11/19 o86 notifications@github.com

**Comment by o86** [comment]
the motivation is not the problem...the work load is :\ however, I push it on my todo-list and let you know if I have something. Please inform me if I would be to late, not that I write code already written.

**Comment by awasthi-vivek** [comment]
It sounds like a very good feature to have... I have been trying to set the "device_country" and "operatorCountry" to different location but the result does not change... I am sure I must be doing something wrong... could you please guide me what are the fields that needs to be changed to change the location?

I am also interested to modify the code and location and lang to the configuration file as suggested.. please let me know the procedure you follow

**Comment by egirault** [comment]
Hi,
I just added localization support (see c463cbe58) but it is only a translation matter. Basically it just enables Google's translation, but will not allow you to see apps available in other countries or mobile operators. It seems those settings are bound to your device during registration. I have to take a closer look at https://github.com/nviennot/android-checkin.

**Comment by awasthi-vivek** [comment]
Thank-you very much for your response. The Android-Checkin looks quite interesting... however I believe that there should be a way to specify the location because when we go roaming then we see the market of the current location.... If the location had been fixed at the time of check-in (registration) then we would see the same market always irrespective of location. What do you think?

In the old project the simOperatorNumeric and operatorNumeric (simOperatorAlpha & operatorAlpha) can be used to change the location.

**Comment by aliemara** [comment]
I am in the US but I want results from India's market. is there any way to do it using this project? Thanks.
